src/ORM/Behavior/TranslateBehavior.php

```diff
@@ -140,7 +143,13 @@ public function setupFieldAssociations($fields, $table, $model, $strategy)
 
         foreach ($fields as $field) {
             $name = $alias . '_' . $field . '_translation';
-
+            $conditions = [
+                $name . '.model' => $fieldConditions['model'],
+                $name . '.field' => $field,
+            ];
+            foreach ($fieldConditions as $fieldName => $fieldValue) {
+                $conditions[$name . '.' . $fieldName] = $fieldValue;
```

Comment:
That's the tricky part, since those aliases are constructed it means the user would need to understand *how* they're constructed. Or am I misunderstanding what you're saying? How were you expecting the conditions to be set? They have to update the fields for the original issue to be satisfied.